src/main/java/com/api/mapper/RocketMapper.java

```diff
@@ -54,4 +56,6 @@ public interface RocketMapper {
     Payloads toDto(SxPayloads payloads);
 
     CompositeFairing toDto(SxCompositeFairing compositeFairing);
+
+    FlickrImages toDto(SxFlickrImages flickrImages);
```

Comment:
so basically on namings you will need to have 3 entity types at this moment: RocketDto (used as response in Controller), RocketSx (Used when fetching the model from api), RocketJpa (used as table_entity in DB)